extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/RevisionTemplateConfig.java

```diff
@@ -0,0 +1,306 @@
+package io.quarkus.kubernetes.deployment;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import io.dekorate.kubernetes.annotation.ImagePullPolicy;
+import io.dekorate.kubernetes.annotation.ServiceType;
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class RevisionTemplateConfig {
```

Comment:
Where is this class being used? I can't seem to find it anywhere else

Reply:
Nowhere, I will remove it!